pkg/api/v1beta3/types.go

```diff
@@ -1051,6 +1051,8 @@ type StatusDetails struct {
 	// The Causes array includes more details associated with the StatusReason
 	// failure. Not all StatusReasons may provide detailed causes.
 	Causes []StatusCause `json:"causes,omitempty" description:"the Causes array includes more details associated with the StatusReason failure; not all StatusReasons may provide detailed causes"`
+	// If specified, the time in seconds before the operation should be retried.
+	RetryAfter int `json:"retryAfter,omitempty" description:"the number of seconds before the client should attempt to retry this operation"`
```

Comment:
How will this interact with the HTTP header field?

My general thinking on that is that we drive behavior off json, but duplicate the info in standard headers (e.g., Etags and If-Match) for the benefit of components/clients that deal with HTTP more generically.